src/file-events.c

```diff
@@ -139,13 +140,14 @@ static __always_inline void enter_mkdir(struct pt_regs *ctx)
     return;
 }
 
-static __always_inline void store_dentry(struct pt_regs *ctx, void *dentry)
+static __always_inline void store_dentry(struct pt_regs *ctx, void *path, void *dentry)
 {
     u64 pid_tgid = bpf_get_current_pid_tgid();
 
     load_event(incomplete_mkdirs, pid_tgid, incomplete_mkdir_t);
     if (event.target_dentry == NULL) {
         event.target_dentry = dentry;
+        event.target_vfsmount = read_field_ptr(path, CRC_PATH_MNT);
```

Comment:
`read_field_ptr` purposefully sets the local warning if the pointer is `NULL`. I think we always expect `vfsmount` not to be NULL so we should emit a warning if we get into a bad state where it is NULL somehow and then exit early out of here.

Reply:
I'll check this in case somehow our offsets led to the lookup failing, but this wouldn't have been called by the kernel without a valid path reference.